Mathlib/FieldTheory/IntermediateField/Adjoin/Basic.lean

```diff
@@ -771,4 +771,40 @@ theorem cardinalMk_adjoin_le {E : Type u} [Field E] [Algebra F E] (s : Set E) :
     #(adjoin F s) ≤ #F ⊔ #s ⊔ ℵ₀ := by
   simpa using lift_cardinalMk_adjoin_le F s
 
+section AdjoinDouble
+
+variable {K L : Type*} [Field K] [Field L] [Algebra K L] {x y : L}
+
+theorem AdjoinDouble.isAlgebraic (hx : IsAlgebraic K x) (hy : IsAlgebraic K y) :
```

Comment:
I'd call this something like `isAlgebraic_adjoin_pair` (cf. [Set.pair_comm](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Set/Insert.html#Set.pair_comm) etc.)